scripts/test-osi-identityd.sh

```
@@ -0,0 +1,1172 @@
+assert_file_exists "$OSI_IDENTITY_RUN_DIR/$SENTINEL_NAME" "sentinel retained after removal failure"
+assert_file_exists "$OSI_IDENTITY_RUN_DIR/$COMPLETION_NAME" "completion marker after removal failure"
+cmp \
+    "$OSI_IDENTITY_RUN_DIR/$SENTINEL_NAME" \
+    "$OSI_IDENTITY_RUN_DIR/$COMPLETION_NAME" || \
+    fail "completion marker does not match the exact sentinel"
+set_now 1001
+identityd_control_tick
+assert_eq "1" \
+    "$(grep -c '^restart$' "$IDENTITYD_TEST_STATE_DIR/service.log")" \
+    "completion marker suppresses repeated restart"
+. "$DAEMON"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)